scripts/gen-ocdb-app.stanza

```diff
@@ -6,14 +6,13 @@ defpackage fig-app :
   import core
   import collections
   import math
-  import esir
-  import esir/utils
-  import esir/gen
-  import esir/repl-lib
-  import ocdb/tests/default-harness
+  import jitx
+  import jitx/gen
+  import jitx/commands
+  import ocdb/defaults
   import ocdb/land-patterns
   import ocdb/generic-components
-  import jitpcb/visualizer
+  import jitx/visualizer
```

Comment:
Have you run this? Do you get shiny svgs?

Reply:
That's the next step. Just modified the imports for now.